tests/libfixmath/tests/01-run.py

```diff
@@ -23,7 +23,7 @@ def expect_unary(child):
 
 
 def expect_binary(child):
-    for _ in range(20):
+    for _ in range(1500):
```

Comment:
This looks strange to me. It seems one needs multiple passes when testing on real hardware ?

Reply:
no, just do the maths, actually there are over 1600 single tests or checks to be done. Its just that on native its very quick and hence doesn't hit the default timeout of pexpect. But on real hardware its way slower hence timeout kicks in.

Reply:
Not sure I'm fully convinced. Do we want to test that all single test/check has the right format ? Why not just increase the timeout value in the `child.expect` call ?

Reply:
IMHO increasing the timeout is suboptimal, to what (reasonable) value do we set the timeout? Because the timeout is highly subjective to the actual board its running on, so its best to set the correct number of expected iterations instead.

Reply:
> it's best to set the correct number of expected iterations instead.

Ok :)